util/remove-bucket.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
+
+"""S3 Bucket Removal
+
+Empties and deletes an AWS S3 bucket.
+"""
+
+import argparse
+import boto3
+import sys
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description=__doc__)
+    parser.add_argument(
+        'bucket_name',
+        help='The name of the bucket to remove')
+    parser.add_argument(
+        '-y', '--yes',
+        action='store_true',
+        help='Skip prompt to verify bucket deletion')
+    parser.add_argument(
+        '--aws-profile',
+        help='AWS user profile to use when making boto3 requests')
+    return parser.parse_args()
+
+
+def verify_bucket(bucket_name):
+    """Prompts the user to verify that the bucket should be deleted"""
+    verify = input(
+        'Please confirm that you want to remove bucket {} (y/n)'
+        .format(bucket_name))
+    if verify != 'y':
+        print('Exiting')
+        sys.exit(1)
+
+
+def empty_bucket(session, bucket_name):
+    """Empties an S3 bucket of all versions, which is required
```

Comment:
Just wondering if this client is event needed?  Would just using the AWS CLI would be better if you don't version your buckets?

Reply:
Reminder, this is largely for buckets created by the toil leader, and its buckets are versioned.

Reply:
By default, and not configurable.